views/layout.hbs

```diff
@@ -22,7 +23,11 @@
     </div>
     <div class="navbar-collapse collapse">
         <ul class="nav navbar-nav navbar-right rb"> 
-          <li><a href="#contact">Pony</a></li>
+          <li><a href="/leaguesearch/soccer">Soccer</a></li>
```

Comment:
Don't you think this would look better with just a "league" selection and then have all the leagues be selectable in a dropdown? I feel like this is making the right side too busy with unneeded stuff.